_posts/batlopez/2016-04-30-Nat'sFinalProj.md

```diff
@@ -0,0 +1,31 @@
+--- 
+layout: post
+author: batlopez
+title: "Nat's Final Data Analysis Project"
+---
+<iframe src="https://trinket.io/embed/python3/49f306fcd8" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+For my data analysis final project, I used UNC course catalog data and wrote a script that would parse through the description in search of a user-generated keyword. As this project is for a work assignment, I initially planned on parsing through data from LibAnalytics that documents courses the Research Hub at Davis Library has supported within the past two years to generate a list of terms that were most frequently found in the descriptions. However, after creating a step by step plan, I decided to break this project into smaller phases; the first of which is this project. 
+
+Ultimately, I think I had two big successes with this project: 
+
+WIN 1: Learning Project Management and Scoping Skills!
+
+The biggest take away from this project was the ability to develop stronger project management and scoping skills. Developing milestones in the beginning of the project was relatively helpful in terms of getting me going and creating a skeletal framework for how to work. However, this was the first time I basically had a dedicated notebook where I broke down the specific tangible steps I needed to take to carry out the aspect of the project I was working on that day. I frequently wrote out what I wanted my program to do and then went back and inserted programming language into my write-up. From that, I was able to pretty consistently generate a list of three to four steps that I wanted to try to focus on and also to prioritize accordingly. There were times where I just generated a list in a random order and was able to reflect and better understand how focusing on one step was necessary before trying to move forward to the next. This was particularly striking when I compared it to my process during the turtle mid-term assignment where I was incredibly overwhelmed and had no ability to just focus on any one part at a time. Instead, I kept frustratingly going back and forth between multiple parts and thus, was never really able to truly get any one thing to work. Suddenly, my project became really small manageable tasks that were immensely rewarding and motivating whenever they were completed. I actually really enjoyed working on this project and TOTALLY want to continue coding now! Thanks Elliott! 
```

Comment:
All in a day's work, ma'am. 🚀 

Seriously, though, keep these skills!  I imposed the framework this time but next time you can impose the framework on yourself just as easily. Project management FTW